src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,35 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from 'react';
+import { Comment } from '../../types/comment';
+import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  commentInfo: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ commentInfo }) => {
+  const { name } = commentInfo;
+  const { email } = commentInfo;
+  const { body } = commentInfo;
```

Comment:
```suggestion
  const {
    name,
    email,
    body,
  } = commentInfo;
```
You can get them all at once